wandb/cli/cli.py

```diff
@@ -993,12 +986,22 @@ def launch_sweep(
     else:
         parsed_sweep_config = parsed_config
 
-    num_workers = num_workers or scheduler_args.get("num_workers", 8)
+    # validate job existence, add :latest alias if not specified
+    if "job" in parsed_sweep_config:
+        if ":" not in parsed_sweep_config["job"]:
+            wandb.termwarn("No alias specified for job, defaulting to 'latest'")
+            parsed_sweep_config["job"] = f"{parsed_sweep_config['job']}:latest"
+        try:
+            api.artifact(parsed_sweep_config["job"], type="job")
+        except Exception:
+            wandb.termerror(traceback.format_exc())
+            return False
+
     scheduler_entrypoint = sweep_utils.construct_scheduler_entrypoint(
         sweep_config=parsed_sweep_config,
         queue=queue,
         project=project,
-        num_workers=num_workers,
+        num_workers=scheduler_args.get("num_workers", 8),
```

Comment:
Why do we want to remove this configurability from this command line? Will we not allow users to call this cli arg directly?

Reply:
@KyleGoyette  yeah its just not important enough to make a CLI param imo. The other cli params are like P1 critical to sweep function, this one is just a tweak....